pandas/tests/sparse/test_series.py

```diff
@@ -1360,4 +1360,4 @@ def test_numpy_func_call(self):
                  'argmin', 'argmax', 'min', 'max']
         for func in funcs:
             for series in ('bseries', 'zbseries'):
-                getattr(np, func)(getattr(self, series))
+                getattr(np, func)(getattr(self, series)._values)
```

Comment:
@lphk92 Can you update this based on the discussion above. So remove the `_values` again, and move argmax/argmin out of the loop to test it separately with catching of the deprecation warnings.